CHANGELOG.md

```diff
@@ -4,6 +4,18 @@
 
 ### 🛑 Breaking changes 🛑
 
+### 🚩 Deprecations 🚩
+
+### 🚀 New components 🚀
+
+### 💡 Enhancements 💡
+
+### 🧰 Bug fixes 🧰
+
+## v0.50.0
+
+### 🛑 Breaking changes 🛑
+
 - `datadogexporter`: Replace HistogramMode defined as string with enum.
```

Comment:
```suggestion
- `datadogexporter`: Replace HistogramMode defined as string with enum (#9589)
```

Reply:
And another one missed below

```
- `exporter/azuremonitor`: Deprecate use of LogRecord.Name as the log envelope category name. There is no replacement (#9258)
```

Reply:
confused, that `azuremonitor` is already under deprecation?

Reply:
Found it, you want to PR number, but that should be checked during review :)

Reply:
Done.